pip/cmdoptions.py

```diff
@@ -114,7 +114,7 @@ def getname(n):
     dest='quiet',
     action='count',
     default=0,
-    help='Give less output.')
+    help='Suppress all output except errors.')
```

Comment:
Maybe be even more explicit ?
cf https://github.com/pypa/pip/blob/master/pip/basecommand.py#L109-L114

With a text similar to the ``--verbose`` option ``Option is additive, and can be used up to 3 times.``